app/models/labware_creators/pcr_cycles_binned_plate_for_t_nano_seq.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module LabwareCreators
+  # This version of the class is specific to the Targeted NanoSeq pipeline.
+  class PcrCyclesBinnedPlateForTNanoSeq < PcrCyclesBinnedPlateBase
+    self.page = 'pcr_cycles_binned_plate_for_t_nano_seq'
+
+    CUSTOMER_FILE_NAME = 'targeted_nano_seq_customer_file.csv'
+
+    # rubocop:disable Metrics/AbcSize
+    def after_transfer!
+      # called as part of the 'super' call in the 'save' method
+      # retrieve child plate through v2 api, using uuid got through v1 api
+      child_v2_plate = Sequencescape::Api::V2.plate_with_custom_includes(CHILD_PLATE_INCLUDES, uuid: child.uuid)
+
+      # update fields on each well with various metadata
+      child_wells_by_location = child_v2_plate.wells.index_by(&:location)
+
+      well_details.each do |parent_location, details|
+        child_well_location = transfer_hash[parent_location]['dest_locn']
+        child_well = child_wells_by_location[child_well_location]
+
+        # NB. this seems to return an array of requests via the api but a single request in tests
+        request = Array(child_well.aliquots.first.request).first
+
+        if request.blank?
+          raise StandardError, "Unable to identify request for child plate well at location #{child_well_location}"
+        end
+
+        # create hash containing the key value pairs we want to store as metadata on the request
+        request_metadata = {
+          'original_plate_barcode' => parent.human_barcode,
+          'original_well_id' => parent_location,
+          'concentration_nm' => details['concentration'],
+          'input_amount_available' => details['input_amount_available'],
+          'input_amount_desired' => details['input_amount_desired'],
+          'sample_volume' => details['sample_volume'],
+          'diluent_volume' => details['diluent_volume'],
+          'pcr_cycles' => details['pcr_cycles'],
+          'hyb_panel' => details['hyb_panel']
+        }
```

Comment:
If the same keys are used for other purposes on the request we need to think about some prefixing or dotted hierarchical namespacing for them.

Reply:
yes, there is some risk there
katy also suggested a 'friendly name' column, so the key can be internal and prefixed etc. but we also have a display name